src/comments.js

```diff
@@ -834,7 +834,33 @@ function printComment(commentPath, options) {
       return "#" + comment.value;
     case "CommentBlock":
     case "Block":
-      return "/*" + comment.value + "*/";
+      const lines = comment.value.split("\n");
+      const isJsDocComment =
+        lines.length > 1 &&
+        lines.slice(0, lines.length - 1).every(line => line.trim()[0] === "*");
+      if (!isJsDocComment) {
+        return "/*" + comment.value + "*/";
+      }
```

Comment:
I'd get rid of this check, I can't think of any kind of multi-line comment where you'd like the lines after the first one to stay at the same position if the first one is shifted.

Reply:
@vjeux I'm still having trouble with the approach that you described.

By the time we get to `printComment`, the `/*` has already been stripped off of `commentPath`, so we can't use it to get the `startIndent`.

I also still can't seem to get `align` to behave properly. Even if I pass in an arbitrary indent value (like `100`), it seems to have no effect on the output :confused:

Reply:
@maxdeviant I haven’t really been following along here, but if I were to make an attempt at fixing the indentation of jsdoc comments, this is how I’d go about it:

1. Split the comment into lines.
2. Trim whitespace from every line.
3. Add a single space at the start of every line except the first.
4. Concatenate all the lines with a hardline between each line.

That should take care of all indentation, shouldn’t it? Or am I missing something?

Reply:
@lydell That is actually the approach I did take. The caveat here is that I needed to only trim the whitespace for JSDoc comments, otherwise regular block comments (that don't start each line with ` * ` would have their indentation broken.

All of the tests are actually passing now, so I believe it is working as intended, just not in the way the @vjeux was describing.